components/shared/format/Identity.vue

```diff
@@ -43,14 +43,14 @@
           >
         </template>
       </span>
-      <template v-if="!hideIdentityPopover">
+      <template v-if="!hideIdentityPopover && !showOnchainIdentity">
```

Comment:
do we even need both of these props?

Reply:
@zhengow just quickly check if they do similar things and maybe we can combine these into one property.

Reply:
<img width="859" alt="image" src="https://user-images.githubusercontent.com/38389586/177722729-0083e615-04d5-4084-bdc4-794feec4d60d.png">
There are lots of components use `Identity.vue`.
I'm worry that if I combine these into one property will cause other bugs.
But if it is necessary, I can do that, but maybe need deeply check.

Reply:
gotcha, thanks for checking